warp/src/constellation/mod.rs

```diff
@@ -47,7 +47,7 @@ impl Stream for ConstellationEventStream {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, serde::Serialize)]
```

Comment:
Lets put this derive behind a cfg_attr for wasm32.

Reply:
as you later suggested on discord, I instead made a copy of this enum, that way it can also hold the error information instead of skipping it on serialization. (Error isn't serializable) It turns out JsError wasn't serializable either, so I used String.